src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -326,7 +327,8 @@ const EntityTableToolbar = ({
         ),
         [OS_CHIP]: (deleted) => setOsFilterValue(xor(osFilterValue, deleted.chips.map(({ value }) => value))),
         [RHCD_FILTER_KEY]: (deleted) => setRhcdFilterValue(onDeleteFilter(deleted, rhcdFilterValue)),
-        [LAST_SEEN_CHIP]: (deleted) => setLastSeenFilterValue(onDeleteFilter(deleted, [lastSeenFilterValue.mark])),
+        [LAST_SEEN_CHIP]: (deleted) =>
+        {setLastSeenFilterValue(onDeleteFilter(deleted, [lastSeenFilterValue.mark])), setStartDate(), setEndDate();},
```

Comment:
nit-pick: this somewhat looks weird :-D Can't we rewrite it to the clearer:

```suggestion
        {
          setLastSeenFilterValue(onDeleteFilter(deleted, [lastSeenFilterValue.mark]));
          setStartDate();
          setEndDate();
        },
```